resources-domain/packages/resources-search-config/tests/convertDocumentResources.test.ts

```diff
@@ -38,6 +38,7 @@ describe('convertIndexDocument', () => {
         { key: 'description', value: 'This is the description', language: '' },
         { key: 'keywords', value: 'keyword1', language: '' },
         { key: 'keywords', value: 'keyword2', language: '' },
+        { key: 'taxonomies/taxonomy1', value: 'taxonomy1', language: '' },
```

Comment:
I think this test is not being run currently.

Reply:
If you move the file from `resources-domain/packages/resources-search-config/tests/convertDocumentResources.test.ts` to  `resources-domain/packages/resources-search-config/tests/unit/convertDocumentResources.test.ts` they should start running